execution.bs

```diff
@@ -4054,171 +4076,120 @@ namespace std::execution {
 
     <pre highlight=c++>
     template&lt;class T>
-    concept <i>movable-value</i> = // exposition only
+    concept <i>movable-value</i> = <i>// exposition only</i>
       move_constructible&lt;decay_t&lt;T>> &&
       constructible_from&lt;decay_t&lt;T>, T>;
     </pre>
 
-## General queries <b>[exec.queries]</b> ## {#spec-execution.queries}
+## Queries <b>[exec.queries]</b> ## {#spec-execution.queries}
 
-### `execution::get_scheduler` <b>[exec.queries.get_scheduler]</b> ### {#spec-execution.receivers.queries.get_scheduler}
+1. A <i>query object</i> is a customization point object that accepts as its
+    first argument a `queryable` object, and for each such invocation that is
+    valid, produces a value of the corresponding property of the object.
 
-1. `execution::get_scheduler` is used to ask an object for its associated scheduler.
+2. Unless otherwise specified, given a queryable object `e`, a query object
+    <code><i>Q</i></code>, and a pack of subexpressions `args`, the value
+    returned by the expression <code><i>Q</i>(e, args...)</code> is valid as
+    long as `e` is valid.
 
-2. The name `execution::get_scheduler` denotes a customization point object. For some subexpression `r`, if the type of `r` is (possibly cv-qualified) `no_env`, then `execution::get_scheduler(r)` is ill-formed. Otherwise, it is expression equivalent to:
+### `queryable` concept <b>[exec.queries.queryable]</b> ### {#spec-execution.queries.queryable}
 
-    1. `tag_invoke(execution::get_scheduler, as_const(r))`, if this expression is well formed.
-
-        * <i>Mandates:</i> The `tag_invoke` expression above is not
-            potentially-throwing and its type satisfies `execution::scheduler`.
-
-    2. Otherwise, `execution::get_scheduler(r)` is ill-formed.
+    <pre highlight="c++">
+    template &lt;class T>
+      concept queryable = destructible&lt;T>;
+    </pre>
 
-3. `execution::get_scheduler()` (with no arguments) is expression-equivalent to `execution::read(execution::get_scheduler)`.
+1. Let `e` be object of type `E`. The type `E` models `queryable` if for each query object <code><i>Q</i></code> and a pack of subexpressions `args`, if <code>requires { <i>Q</i>(e, args...) }</code> is `true` then <code><i>Q</i>(e, args...)</code> is well-formed.
```

Comment:
I made some qualification here, to replace "well formed" to "meets semantics requirements imposed by Q". I think that would be clener.